onnx/defs/experiments/defs.cc

```diff
@@ -106,28 +106,28 @@ NOTE: Currently, it supports data type of float, int32, int64, and bool.
     .Attr(
         "value",
         "The value for the elements of the output tensor.",
-        AttributeProto::FLOAT)
+        0.0f)
     .Attr(
         "dtype",
         "The data type for the elements of the output tensor."
         "Strictly must be one of the types from DataType enum in TensorProto.",
-        AttributeProto::INT)
+        static_cast<int64_t>(1))
     .Attr(
         "shape",
         "The shape of the output tensor."
         "Cannot set the shape argument and pass in an input at the same time.",
-        AttributeProto::INTS)
+        std::vector<int64_t>{})
```

Comment:
This should be a required field.

Reply:
changing it to "float" will break the test. Keep it and fix it in separate PR as well as fixing test failures. Sounds good?